index.js

```diff
@@ -0,0 +1,125 @@
+const listItems = document.querySelector('.list-items');
+const navContainer = document.querySelector('.nav-container');
+const hamburger = document.querySelector('.hamburger-container');
+const bars = document.querySelector('.fa-bars');
+const closeIcon = document.querySelector('.fa-times');
+
+function toggleMenuBar() {
+  if (listItems.classList.contains('show-items')) {
+    listItems.classList.remove('show-items', 'nav-items');
+    closeIcon.style.display = 'none';
+    navContainer.style.display = 'none';
+    bars.style.display = 'block';
+  } else {
+    listItems.classList.add('show-items', 'nav-items');
+    closeIcon.style.display = 'block';
+    navContainer.style.display = 'block';
+    bars.style.display = 'none';
+  }
+}
+
+hamburger.addEventListener('click', toggleMenuBar);
+closeIcon.addEventListener('click', toggleMenuBar);
+
+const speakersObj = [
+  {
+    image: 'images/Winnie.png',
+    name: 'Winnie Mwango',
+    status: 'Frontend Developer at Google',
+    description:
+      'Outstanding Speaker with more than 5 years of experience in Fronted Development.',
+  },
+  {
+    image: 'images/Paul.png',
+    name: 'Paul Onchera',
+    status: 'Software Developer at Microsoft',
+    description:
+      'Great Speaker with more than 6 years of experience in Software Development.',
+  },
+  {
+    image: 'images/Daisy.png',
+    name: 'Daisy Monroe',
+    status: 'Backend Developer at AirBnB',
+    description:
+      'Great Speaker with more than 6 years of experience in Backend Development.',
+  },
+  {
+    image: 'images/Shantel.png',
+    name: 'Daisy Monroe',
+    status: 'Backend Developer at AirBnB',
+    description:
+      'Great Speaker with more than 6 years of experience in Backend Development.',
+  },
+];
```

Comment:
- [ ] [Optional] Kindly be noticed that you should add six individual articles, you only added fout.